.github/workflows/rust.yaml

```diff
@@ -0,0 +1,72 @@
+name: Rust
+
+on:
+  push:
+    branches:
+      - "main"
+    tags:
+      - "v[0-9]+.[0-9]+.[0-9]+*"
+  pull_request:
+
+jobs:
+  lint:
+    name: Format and lint
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          submodules: recursive
+
+      - uses: actions-rs/toolchain@v1
+        with:
+          profile: minimal
+          toolchain: stable
+          override: true
+          components: rustfmt, clippy
+
+      - uses: actions-rs/cargo@v1
+        name: Format Check
+        with:
+          command: fmt
+          args: -- --check
+
+      - uses: actions-rs/cargo@v1
+        name: Clippy
+        with:
+          command: clippy
+          args: --all-features -- -D warnings
+
+  test:
+    name: Tests
+    strategy:
+      matrix:
+        include:
+          - os: ubuntu-18.04
+            binary_target: x86_64-unknown-linux-musl
+          - os: ubuntu-18.04
+            binary_target: x86_64-unknown-linux-gnu
+          - os: macos-10.15
+            binary_target: x86_64-apple-darwin
+          - os: windows-2019
+            binary_target: x86_64-pc-windows-msvc
+    runs-on: ${{ matrix.os }}
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          submodules: recursive
+
+      # required for bindgen on Windows
+      - name: Install LLVM and Clang
+        uses: KyleMayes/install-llvm-action@v1
+        if: matrix.os == 'windows-2019'
+        with:
+          version: 10.0
```

Comment:
I'm not sure how long this action takes to run but llvm is already enabled on the windows runners just not exposed via LIBCLANG_PATH. You could try setting this environment variable to this path:
`LIBCLANG_PATH: "C:/Program Files (x86)/Microsoft Visual Studio/2019/Enterprise/VC/Tools/Llvm/x64/bin"`

Reply:
Worked like a charm!